HelloFlask/views.py

```diff
@@ -1,31 +1,44 @@
-from flask import Flask
-from flask import render_template
+from datetime import datetime
+from re import match
+
+from flask import Flask, render_template
+
 from HelloFlask import app
```

Comment:
I would go for `hello_flask` since `snake_case` names are more common for most python packages (some old stdlib packages aside, but unfortunately many older parts of stdlib aren't the best example for naming anyway)

Reply:
Yes, good point given that I made that piece a module and not just a folder name.